dummydata.sql

```diff
@@ -1,10 +1,10 @@
 -- Insert dummy data for User table
 INSERT INTO `User` (`username`, `password`, `email`, `phoneNumber`, `userType`) VALUES
-('cyclelover', SHA2('password1', 256), 'cyclelover@example.com', '1234567890', 'buyer'),
-('bikeenthusiast', SHA2('password2', 256), 'bikeenthusiast@example.com', '2345678901', 'seller'),
-('urbanrider', SHA2('password3', 256), 'urbanrider@example.com', '3456789012', 'buyer'),
-('mountainmaverick', SHA2('password4', 256), 'mountainmaverick@example.com', '4567890123', 'seller'),
-('trailtamer', SHA2('password5', 256), 'trailtamer@example.com', '5678901234', 'buyer');
+('cyclelover', 'password1', 'cyclelover@example.com', '1234567890', 'buyer'),
```

Comment:
if we change it to this then it won't be saved to the SQL database in a hashed way

Reply:
Yes, but just for these five dummy users. The passwords of new users will be hashed. 
I didn't want to prehash them because I wanted us to be able to see the passwords of the dummy users for testing purposes. 
But to be fair we can see it in the dummydata.sql file if need be; do you think it's a better idea to prehash them?

Reply:
yeah bc then we can log in as the dummy users :))

Reply:
easier to test some things if we have a few we can log in as and e.g try and outbid each other